packages/claim-cli/src/utils/get-private-key.ts

```diff
@@ -0,0 +1,95 @@
+import { select } from '@inquirer/prompts';
+import * as crypto from '@liskhq/lisk-cryptography';
+import { Mnemonic, HDNodeWallet, Wallet } from 'ethers';
+import { remove0x } from './prefix';
+import { getInput, getPassword } from './get-prompts';
+
+enum SecretType {
+	MNEMONIC,
+	PRIVATE_KEY,
+	JSON,
+}
+
+const getSecretType = (wallet: string) =>
+	select({
+		message: `Secret Type for ${wallet}`,
+		choices: [
+			{ name: 'Mnemonic', value: SecretType.MNEMONIC },
+			{ name: 'Private Key', value: SecretType.PRIVATE_KEY },
+		],
+	});
+
+export async function getLSKPrivateKeyFromMnemonic(): Promise<Buffer> {
+	const mnemonic = await getInput({ message: 'Your Mnemonic' });
+	if (!Mnemonic.isValidMnemonic(mnemonic)) {
+		console.log('Invalid Mnemonic, please check again.');
+		return process.exit(1);
+	}
+
+	const path = await getInput({ message: 'Path', default: "m/44'/134'/0'" });
+
+	return crypto.ed.getPrivateKeyFromPhraseAndPath(mnemonic.trim(), path);
+}
+
+export async function getLSKPrivateKeyFromString(): Promise<Buffer> {
+	const privKey = await getInput({
+		message: 'Your Private Key',
+	});
```

Comment:
this should be getting password not to leave the trace on the CLI. Same for Mnemonic